### DISCUSSION TITLE
How to prevent dragover of a children control from triggering when doing a dragdrop of Parent Control?

### DISCUSSION THREAD

**Discussion by WileyNg** [picoe/Eto#2345] [General]
I have a Parent Control that has dragdrop method. This parent Control contains another child that also has DragDrop.

The problem is, when I dragover the parent Control over the child, it dragdrops for the parent is discontinued, until the mouse moves out of the child.

How to prevent the child from triggering the DragOver/DragEnter Event  when the parent is dragging?

**Comment by WileyNg** [comment]
Maybe to better rephrase my question:
How to prevent DragLeave event from triggering when it is dragged over another Control that has OnDragOver override?

**Reply by WileyNg** [reply]
@cwensley 
is it possible you could shed some lights on this?

**Comment by cwensley** [comment]
Not quite sure how one would do this.. you can't prevent child controls from getting events..  I think you could either disable them when the DragEnter begins, or perhaps create an overlay window that accepts the drop instead of the child controls.
